src/ikob/datasource.py

```diff
@@ -51,33 +65,35 @@ def read_parking_times(config):
     parking_time_path = pathlib.Path(config_skims.get("parkeerzoektijden_bestand", segs_dir / "Parkeerzoektijd.csv"))
 
     if parking_time_path.exists():
-        logging.info("Reading parking times: '%s'", parking_time_path)
+        logging.info("Reading parking times from: '%s'", parking_time_path)
         return utils.read_csv_int(parking_time_path)
 
-    urbanisation_path = segs_dir / "Stedelijkheidsgraad.csv"
-    assert urbanisation_path.exists(), (
+    urbanization_path = segs_dir / "Stedelijkheidsgraad.csv"
+    assert urbanization_path.exists(), (
         "Missing both Parkeerzoektijden, Stedelijkheidsgraad files.Parkeerzoektijden file cannot be generated."
     )
 
     msg = "Generating parking times from '%s'"
-    logger.info(msg, urbanisation_path)
-    urbanisation_grade = utils.read_csv_int(urbanisation_path)
-    return urbanisation_grade_to_parking_times(urbanisation_grade)
+    logger.info(msg, urbanization_path)
+    urbanization_grade = utils.read_csv_int(urbanization_path)
+    return urbanization_grade_to_parking_times(urbanization_grade)
 
 
 class SkimsSource:
     """A data provider for skims files."""
 
     def __init__(self, skims_dir: pathlib.Path | str):
+        if skims_dir == "":
+            raise DataSourceError("Skims source initialized with empty skims dir")
         self.skims_dir = pathlib.Path(skims_dir)
 
-    def read(self, id: str, dagsoort: str, type_caster=float):
+    def read(self, id: str, dag_deel: str, type_caster=float):
```

Comment:
```suggestion
    def read(self, id: str, dagdeel: str, type_caster=float):
```

_Dagdeel_ is one word.